tools/tsp-client/src/index.ts

```diff
@@ -172,8 +172,18 @@ async function syncTspFiles(outputDir: string, localSpecRepo?: string) {
     await removeDirectory(cloneDir);
   }
 
-  const emitterPath = joinPaths(repoRoot, "eng", "emitter-package.json");
-  await cp(emitterPath, joinPaths(srcDir, "package.json"), { recursive: true });
+  try {
+    const emitterLockPath = joinPaths(repoRoot, "eng", "emitter-package-lock.json");
+    await cp(emitterLockPath, joinPaths(srcDir, "package-lock.json"), { recursive: true });
+  } catch (err) {
+    Logger.debug("emitter-package-lock.json not found, will look for emitter-package.json");
```

Comment:
There could be other reasons (maybe not too common) why copy failed, for example if the disk is full or for some reason `srcDir` doesn't exist. If users fall into those situations, this log may be missleading

Reply:
That's true, I updated the log to include the original error but still wont fail in this case if we have an emitter-package.json. If the issue happens with emitter-package.json then we will fail